docs/administration/protect-sensitive-configuration.md

```diff
@@ -41,3 +41,48 @@ Only users with `Allow modification of sensitive configuration` capability enabl
 
 ![role_allow_modification_sensitive_conf.png](assets%2Frole_allow_modification_sensitive_conf.png)
 
+## Recommended approach to give yourself access to danger zone
+
+1. With a user having the capability “Manage credentials” or “Bypass all capbilities”, go into parameters/security
+2. Create a new role called “Danger Zone Administration”.
+3. Give it the capability **“**Allow modification of sensitive configuration”
+4. Create a group “Danger Zone Administrator”
+5. Add a user of your choice (yourself for instance): at this stage you should be all good
```

Comment:
5. Add a user of your choice (yourself for instance): at this stage, you should be able to manage any area under danger zone.